docs/src/index.md

```diff
@@ -1,4 +1,54 @@
-# ClimaLand.jl
+```@eval
+using Pkg
+io = IOBuffer()
+v = Pkg.installed()["ClimaLand"]
+print(io, """
+    # ClimaLand.jl Documentation (v$(v))
+
+    """)
+import Markdown
+Markdown.parse(String(take!(io)))
+```
```

Comment:
I think that this adds a chunk of code to be maintained but very little value (the version number).

Reply:
I agree, and removed it